common/src/main/java/gov/cms/ab2d/common/dto/Mapping.java

```diff
@@ -0,0 +1,16 @@
+package gov.cms.ab2d.common.dto;
```

Comment:
This class makes sense only if `ModelMapper` is expensive (slow) to create. Furthermore, `ModelMapper` must be thread-safe to be used as a Singleton.

Unless instantiating `ModelMapper` is really slow, I'd say let's drop this class and use `new ModelMapper` wherever it is needed.

Reply:
I think with the recent changes that require mappings and converters it's a little more expensive to create so it makes sense to have this class be a bean since it requires other components that we will use. It's thread safe since it's using concurrent hash maps where it keeps state and other places that keep state are done just at the object creation time.